server.js

```diff
@@ -124,8 +125,10 @@ async function work(body) {
     findPotentialReviewers: true,
     actions: ['opened'],
     skipAlreadyAssignedPR: false,
+    delayed: false,
+    delayedUntil: "3d",
     assignToReviewer: false,
-    skipTitle: "",
+    skipTitle: ""
```

Comment:
(if you can fix this double quote as well would be nice. Also, please restore the trailing comma